### PR TITLE
Prepare Spring App Configuration March release

### PR DESCRIPTION
**Pull request by saragluna**
As title.

### CODE REVIEW

**Review by backwind1233 — APPROVED** [review]
LGTM

File: sdk/appconfiguration/azure-spring-cloud-appconfiguration-config-web/CHANGELOG.md [hunk 1]
```diff
@@ -1,14 +1,11 @@
...
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+### Dependency Upgrades
+- Regular updates for Azure SDK dependency versions.
+- Upgrade external dependencies' version according to [spring-boot-dependencies:2.6.3](https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/2.6.3/spring-boot-dependencies-2.6.3.pom).
```
Comment (chenrujun) [inline sdk/appconfiguration/azure-spring-cloud-appconfiguration-config-web/CHANGELOG.md:8]: And `spring-cloud-dependencies:2021.0.1`.
Refs: https://github.com/Azure/azure-sdk-for-java/pull/27147/files

File: sdk/appconfiguration/azure-spring-cloud-appconfiguration-config/CHANGELOG.md [hunk 2]
```diff
@@ -1,11 +1,16 @@
...
+This release is compatible with Spring Boot 2.5.0-2.5.11, 2.6.0-2.6.5.
 
+### Features Added
 * Added refresh interval parameter to `spring.cloud.azure.appconfiguraiton` to force refreshes on a given interval. Can be used to make sure secrets are kept up to date.
 * Added BackoffTimeCalculator, which sets the next refresh period to sooner if a refresh fails.
 
-### Other Changes
+### Dependency Upgrades
+- Regular updates for Azure SDK dependency versions.
+- Upgrade external dependencies' version according to [spring-boot-dependencies:2.6.3](https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/2.6.3/spring-boot-dependencies-2.6.3.pom).
```
Comment (chenrujun) [inline sdk/appconfiguration/azure-spring-cloud-appconfiguration-config/CHANGELOG.md:12]: Same here.

File: sdk/appconfiguration/azure-spring-cloud-feature-management-web/CHANGELOG.md [hunk 3]
```diff
@@ -1,14 +1,11 @@
...
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+### Dependency Upgrades
+- Regular updates for Azure SDK dependency versions.
+- Upgrade external dependencies' version according to [spring-boot-dependencies:2.6.3](https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/2.6.3/spring-boot-dependencies-2.6.3.pom).
```
Comment (chenrujun) [inline sdk/appconfiguration/azure-spring-cloud-feature-management-web/CHANGELOG.md:8]: Here.

File: sdk/appconfiguration/azure-spring-cloud-appconfiguration-config-web/CHANGELOG.md [hunk 4]
```diff
@@ -1,14 +1,11 @@
 # Release History
 
-## 2.5.0-beta.1 (Unreleased)
+## 2.5.0 (2022-03-28)
+This release is compatible with Spring Boot 2.5.0-2.5.11, 2.6.0-2.6.5.
```
Comment (chenrujun) [inline sdk/appconfiguration/azure-spring-cloud-appconfiguration-config-web/CHANGELOG.md:4]: How about keeping same style with 4.0 change log?

File: sdk/appconfiguration/azure-spring-cloud-feature-management/CHANGELOG.md [hunk 5]
```diff
@@ -1,8 +1,14 @@
...
+## 2.4.0 (2022-03-28)
+This release is compatible with Spring Boot 2.5.0-2.5.11, 2.6.0-2.6.5.
 
-Updated PercentageFilter to support Strings and Doubles
+### Features Added
+- Updated PercentageFilter to support Strings and Doubles.
+
+### Dependency Upgrades
+- Regular updates for Azure SDK dependency versions.
+- Upgrade external dependencies' version according to [spring-boot-dependencies:2.6.3](https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/2.6.3/spring-boot-dependencies-2.6.3.pom).
```
Comment (chenrujun) [inline sdk/appconfiguration/azure-spring-cloud-feature-management/CHANGELOG.md:11]: Here.

File: sdk/appconfiguration/azure-spring-cloud-starter-appconfiguration-config/CHANGELOG.md [hunk 6]
```diff
@@ -1,14 +1,11 @@
...
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+### Dependency Upgrades
+- Regular updates for Azure SDK dependency versions.
+- Upgrade external dependencies' version according to [spring-boot-dependencies:2.6.3](https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/2.6.3/spring-boot-dependencies-2.6.3.pom).
```
Comment (chenrujun) [inline sdk/appconfiguration/azure-spring-cloud-starter-appconfiguration-config/CHANGELOG.md:8]: Here.

**Review by chenrujun — CHANGES_REQUESTED** [review]

File: sdk/appconfiguration/azure-spring-cloud-appconfiguration-config-web/CHANGELOG.md (hunk 4 above)
Comment (saragluna) [reply sdk/appconfiguration/azure-spring-cloud-appconfiguration-config-web/CHANGELOG.md:4]: ok

**Review by chenrujun — APPROVED** [review]
LGTM.

**Pull request merged**